feature-libs/order/root/feature-name.ts

```diff
@@ -0,0 +1,2 @@
+export const ORDER_FEATURE = 'cartOrder';
+export const ORDER_CORE_FEATURE = 'cartOrderCore';
```

Comment:
There is `cart` in the feature name, is that intentional ?

Reply:
no, I forgot to change it :)